src/Bridges/Nette/Bridge.php

```diff
@@ -33,6 +33,7 @@ public static function initialize(): void
 
 	public static function renderLatteError(?\Throwable $e): ?array
 	{
+		assert(\class_exists(Latte\CompileException::class));
```

Comment:
it's better to add `latte/latte` as a dev dependency

Reply:
Added.